apidef/oas/operation.go

```diff
@@ -163,6 +165,83 @@ func (s *OAS) fillPathsAndOperations(ep apidef.ExtendedPathsSet) {
 	s.fillDoNotTrackEndpoint(ep.DoNotTrackEndpoints)
 	s.fillRequestSizeLimit(ep.SizeLimit)
 	s.fillRateLimitEndpoints(ep.RateLimit)
+	s.fillMockResponsePaths(s.Paths, ep)
+}
+
+func (s *OAS) fillMockResponsePaths(paths openapi3.Paths, ep apidef.ExtendedPathsSet) {
+	if paths == nil {
+		paths = make(openapi3.Paths)
+	}
+
+	for _, mockResponse := range ep.MockResponse {
+		path := mockResponse.Path
+		method := mockResponse.Method
+
+		if paths[path] == nil {
+			paths[path] = &openapi3.PathItem{}
+		}
+
+		statusCode := strconv.Itoa(mockResponse.Code)
+
+		contentType := "text/plain"
+		if ct, ok := mockResponse.Headers["Content-Type"]; ok && ct != "" {
+			contentType = ct
+		} else if ct, ok := mockResponse.Headers["content-type"]; ok && ct != "" {
+			contentType = ct
+		}
+
+		// Detect JSON content
+		if mockResponse.Body != "" {
+			var js interface{}
```

Comment:
```suggestion
			var js interface{}
```

Could this be json.RawMessage?

Reply:
Let fix that, thanks

Reply:
Fixed it, PTAL